yaas/config.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+
+# These default configs are overriden based on
+# command line arguments or environment variables
+
+server_url = None   # Must be overriden
+requests_kwargs = {
```

Comment:
Name this something more succinct. We're gonna be using it all the time. How about `ropts`?